INSTALL.md

```diff
@@ -19,29 +17,54 @@ docker run bystro bystro-annotate.pl #Annotate
 docker run bystro bystro-build.pl #Build
 ```
 
-## Installation on RPM-based distros
+# Direct (non-Docker) installation
 
-###### (Fedora, Redhat, Centos, openSUSE, Mandriva)
+There are 2 components to Bystro:
+ 1. The Bystro annotator: a Perl program accessed through the command line (via bin/bystro-*)
+ 2. The Bystro Python package: where the rest of Bystro's functionality lives (statistics, proteomics, etc).
 
-1.  `git clone https://github.com/akotlar/bystro.git && cd bystro && source ./install-rpm.sh`
+## Installing the Bystro annotator (Perl/cli)
+
+##### (Fedora, Redhat, Centos, openSUSE, Mandriva)
 
-## Installation on MacOS
+1.  `git clone https://github.com/akotlar/bystro.git && cd bystro && source ./install-rpm.sh`
```

Comment:
here and throughout, do we want to clone from bystrogenomics rather than from your personal branch?

Reply:
Oops good catch